### DISCUSSION TITLE
Deployment in medium mode

### DISCUSSION THREAD

**Discussion by Mikaela** [uBlockOrigin/uBlock-issues#3116] [Ideas] [answered]
I would like to deploy µBlock Origin in medium mode by default (like [this Reddit user](https://www.reddit.com/r/uBlockOrigin/comments/144993e/deployment_in_medium_mode_on_firefox/)) so whenever I create a new Firefox profile, I would start with scripts blocked without having to manually follow the instructions in the wiki. I also don't want to reset the settings every time I start the browser on previously configured profile, which I understood from the previously linked Reddit to be the case.

- https://github.com/gorhill/uBlock/wiki/Deploying-uBlock-Origin
  - https://github.com/gorhill/uBlock/wiki/Deploying-uBlock-Origin:-configuration
- https://github.com/gorhill/uBlock/wiki/Blocking-mode:-medium-mode

Currently it seems that the easiest option is to deploy NoScript and deal with its defaults.

**Comment by ghost** [comment, marked as answer]
You can create a template by setting how you like it and then save the backup.

The original idea was that dynamic filtering should be set by the user - no preconfigured lists - no idea if gorhill change his mind.

**Comment by Mikaela** [comment]
I had forgotten this discussion, but the solution I eventually ended up to is having `https://github.com/gorhill/uBlock/wiki/Dynamic-filtering:-quick-guide` in the `|` separated `OverrideFirstRunPage` so I will remember to enable it and to make it a few steps less `"userSettings": [[ "advancedUserEnabled", "true"]]` so I only have to click "more" a couple of times and then make 3rd party scripts red.
